### DISCUSSION TITLE
有没有微信群或者qq群啊

### DISCUSSION THREAD

**Discussion by cxjhk** [wind-c/comqtt#19] [General]
我们项目准备从gmqtt转到这个项目，但是不知道是否有大量生产案例没

**Comment by wind-c** [comment]
> 我们项目准备从gmqtt转到这个项目，但是不知道是否有大量生产案例没

别急，正在进行2.0版本开发，提升消息吞吐和集群稳定性。

**Comment by wind-c** [comment]
微信群
![mmqrcode1678775352690.png](https://user-images.githubusercontent.com/17872481/224914812-e58a8107-0115-44db-a797-c17ee2e7599b.png)
